server/routes/bookedtoday/unexpectedArrivals/arrivals/noExistingRecordsFoundController.ts

```diff
@@ -0,0 +1,12 @@
+import type { RequestHandler } from 'express'
+import type { ExpectedArrivalsService } from '../../../../services'
+
+export default class NoExistingRecordsFoundController {
+  public constructor(private readonly expectedArrivalsService: ExpectedArrivalsService) {}
+
```

Comment:
I think maybe best not to pass services down to controllers until we need them there